### PR TITLE
Feature/view applicant profile screen

### PR DESCRIPTION
**Pull request by WalterGoldenOwl**
Description Pull Request:

## Todo
- Apply UI and logic view personal profile screen 

## Task
Link of task: 

1. https://trello.com/c/KYmDjyIg
2. https://trello.com/c/kBCClnzb

## Screenshot
Screenshot_1698028189
Screenshot_1698028193
Screenshot_1698028196
Screenshot_1698028199

## Video

https://github.com/dustin-goldenowl/jobspot/assets/144195318/7e780c1a-8497-45c8-aa05-53608bd2e56b

### CODE REVIEW

File: lib/src/presentations/view_applicant_profile/data/repositories/view_applicant_profile_repository_impl.dart [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+@LazySingleton(as: ViewApplicantProfileRepository)
+class ViewApplicantProfileRepositoryImpl
+    extends ViewApplicantProfileRepository {
+  @override
+  Future<DataState<List<WorkExperienceEntity>>> getWorkExperience(
+      String uid) async {
+    try {
+      final response = await FirebaseFirestore.instance
+          .collection("workExperiences")
```
Comment (dustin-goldenowl) [inline lib/src/presentations/view_applicant_profile/data/repositories/view_applicant_profile_repository_impl.dart:27]: Move to XCollection. workExperiences

File: lib/src/presentations/view_applicant_profile/data/repositories/view_applicant_profile_repository_impl.dart [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    } catch (e) {
+      return DataFailed(e.toString());
+    }
+  }
+
+  @override
+  Future<DataState<List<EducationEntity>>> getEducation(String uid) async {
+    try {
+      final response = await FirebaseFirestore.instance
+          .collection("educations")
```
Comment (dustin-goldenowl) [inline lib/src/presentations/view_applicant_profile/data/repositories/view_applicant_profile_repository_impl.dart:44]: The same above

File: lib/src/presentations/view_applicant_profile/data/repositories/view_applicant_profile_repository_impl.dart (hunk 1 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/view_applicant_profile/data/repositories/view_applicant_profile_repository_impl.dart:27]: I fixed it in new branch and new PR

File: lib/src/presentations/view_applicant_profile/data/repositories/view_applicant_profile_repository_impl.dart (hunk 2 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/view_applicant_profile/data/repositories/view_applicant_profile_repository_impl.dart:44]: I fixed it in new branch and new PR

**Pull request merged**